### PR TITLE
Added pick_place demo and simulation env

### PR DESCRIPTION
**Pull request by hanhongsun**
https://github.com/fetchrobotics/motion_planning/issues/134
Adjusted the parameter value to avoid over approaching. 
Created new yaml file for simple_grasping (removed invalid parameter setting from the launch file)
Added more poses for placing instead of 4.
Merged the pick_place version of demo into the palyground demo. Fixed the problems above for both.
Everything was fully tested.

### CODE REVIEW

File: fetch_gazebo/scripts/prepare_simulated_robot.py [hunk 1]
```diff
@@ -27,6 +27,7 @@
 # THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 # Author: Michael Ferguson
+# Author: Di Sun
```
Comment (mikeferguson) [inline fetch_gazebo/scripts/prepare_simulated_robot.py]: I don't see any other changes in this file?

File: fetch_gazebo/CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,11 @@
 Changelog for package fetch_gazebo
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.7.0 (2015-10-21)
```
Comment (mikeferguson) [inline fetch_gazebo/CHANGELOG.rst]: please revert all changes to this file -- this is done as part of the debian release

File: fetch_gazebo_demo/CHANGELOG.rst [hunk 3]
```diff
@@ -2,6 +2,13 @@
 Changelog for package fetch_gazebo_demo
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.7.0 (2015-10-21)
```
Comment (mikeferguson) [inline fetch_gazebo_demo/CHANGELOG.rst]: revert these changes

File: fetch_gazebo_demo/launch/demo.launch [hunk 4]
```diff
@@ -12,9 +12,7 @@
   <node name="basic_grasping_perception" pkg="simple_grasping" type="basic_grasping_perception" >
     <!-- finger tips are 195mm from wrist_roll_link
```
Comment (mikeferguson) [inline fetch_gazebo_demo/launch/demo.launch]: these comments should be removed (since the parameters are no longer here) -- consider adding them to the YAML

File: fetch_gazebo_demo/launch/pick_place_demo.launch [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  <include file="$(find fetch_gazebo_demo)/launch/fetch_nav.launch" />
+
+  <!-- Start MoveIt -->
+  <include file="$(find fetch_moveit_config)/launch/move_group.launch" >
+    <arg name="info" value="true"/><!-- publish grasp markers -->
+  </include>
+
+  <!-- Start Perception -->
+  <node name="basic_grasping_perception" pkg="simple_grasping" type="basic_grasping_perception" >
+    <!-- finger tips are 195mm from wrist_roll_link
```
Comment (mikeferguson) [inline fetch_gazebo_demo/launch/pick_place_demo.launch]: all of these comments are irrelevant -- remove them

**Comment by mikeferguson** [comment]
@hanhongsun please squash these commits so changes to changelog go away completely

File: fetch_gazebo/CMakeLists.txt [hunk 6]
```diff
@@ -49,6 +49,7 @@ install(
 
 install(
   PROGRAMS scripts/prepare_simulated_robot.py
+  PROGRAMS scripts/prepare_simulated_robot_pick_place.py
```
Comment (dirk-thomas) [inline fetch_gazebo/CMakeLists.txt:52]: This line fails with a CMake error on install. The second `PROGRAMS` keyword needs to be removed.

http://build.ros.org:8080/view/Idev/job/Idev__fetch_gazebo__ubuntu_trusty_amd64/2/console#console-section-0
